core/src/utils/animation/animation-utils.ts

```diff
@@ -69,6 +85,7 @@ export const getStyleContainer = (element: HTMLElement) => {
 
 export const createKeyframeStylesheet = (keyframeName: string, keyframeRules: string, element: HTMLElement): HTMLElement => {
   const styleContainer = getStyleContainer(element);
+  const keyframePrefix = getAnimationPrefix(element);
```

Comment:
```suggestion
```

Making the first suggested change means that we can just grab the value of `animationPrefix`.